### ISSUE TITLE
Use API dataset names in overview table

### ISSUE THREAD

**Issue by hagenw** [soundata/soundata#157]
It is a very nice idea to have an overview table of all supported datasets as provided at https://soundata.readthedocs.io/en/latest/source/quick_reference.html

What might be even more convinient would be to use the same dataset names in the table and in the library.
E.g. when looking at 

![image](https://github.com/soundata/soundata/assets/173624/8c677f84-7f83-4594-b7c3-79b56efe290b)

you might try:

```python
>>> import soundata
>>> dataset = soundata.initialize("TAU SSE 2019")
...
ValueError: Invalid dataset TAU SSE 2019
```

instead you need to do:

```python
dataset = soundata.initialize("tau2019sse")
```

It would be great, if a user could just copy and paste the name from the table.

**Comment by hagenw** [comment]
It's even more complicated for

![image](https://github.com/soundata/soundata/assets/173624/fee223b8-6dbb-47c1-9d2d-9eb6139a53d2)

where the dataset name is `marco`.

**Comment by genisplaja** [comment]
Hello @hagenw, thank you very much for your review. We are working on the raised points, and we will be getting back to you soon. About this one, we agree with your point here. Our preferred solution would be to add an extra column to the dataset table indicating the API dataset name (e.g. `marco`) alongside its official formatted name (e.g. `3D-MARCo`). Therefore, we would have the complete dataset name for more information, while indicating the API dataset id to use to initialize the dataset, as some dataset ids ay skip some important parts of the name. What do you think about that? Thanks!

**Comment by tanmayy24** [comment]
PR: https://github.com/soundata/soundata/pull/167

**Comment by guillemcortes** [comment]
Hi @hagenw, we've merged #167 which addresses this issue. You can check how it looks here: https://soundata.readthedocs.io/en/latest/source/quick_reference.html
